jsonnet/kube-prometheus/components/node-exporter.libsonnet

```
@@ -175,8 +175,8 @@ function(params) {
         '--no-collector.wifi',
         '--no-collector.hwmon',
         '--collector.filesystem.ignored-mount-points=^/(dev|proc|sys|var/lib/docker/.+|var/lib/kubelet/pods/.+)($|/)',
-        '--collector.netclass.ignored-devices=^(veth.*)$',
-        '--collector.netdev.device-exclude=^(veth.*)$',
+        '--collector.netclass.ignored-devices=^(veth.*|[a-z0-9]{15}@if\\d+)$',
```

```suggestion
        '--collector.netclass.ignored-devices=^([a-z0-9]+@if\\d+)$',
```

Would it be possible to do the above? It looks like veth devices in kubernetes follow the above pattern, so we might have a generic exclusion for all platforms and not solely for OpenShift.

---

We also might want to add a comment why this particular regex is ignored as it isn't obvious at the first glance.